geode-core/src/distributedTest/java/org/apache/geode/internal/cache/TestDelta.java

```diff
@@ -62,10 +79,18 @@ public boolean hasDelta() {
     return hasDelta;
   }
 
+  @Override
+  public boolean getForceRecalculateSize() {
+    // new Exception("RINGLES - getForceRecalculateSize Called: " +
+    // forceRecalculateSize).printStackTrace();
```

Comment:
This comment should be removed.

Reply:
If so, I'll go ahead and remove the "DAN" ones, too.